CONTRIBUTING.md

```diff
@@ -16,46 +16,102 @@ In addition to Github issues, we also utilize a feedback site that helps our pro
+The DCO is an attestation attached to every contribution made by every developer. In the commit message of the contribution, the developer simply adds a Signed-off-by statement and thereby agrees to the DCO, which you can find below or at <http://developercertificate.org/>.
 
-You can complete our [Individual CLA](https://supermarket.chef.io/icla-signatures/new) online. If you're contributing on behalf of your employer and they retain the copyright for your works, have your employer fill out our [Corporate CLA](https://supermarket.chef.io/ccla-signatures/new) instead.
+```
+Developer's Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+(a) The contribution was created in whole or in part by me and I
+    have the right to submit it under the open source license
+    indicated in the file; or
+
+(b) The contribution is based upon previous work that, to the
+    best of my knowledge, is covered under an appropriate open
+    source license and I have the right under that license to   
+    submit that work with modifications, whether created in whole
+    or in part by me, under the same open source license (unless
+    I am permitted to submit under a different license), as
+    Indicated in the file; or
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including
+    all personal information I submit with it, including my
+    sign-off) is maintained indefinitely and may be redistributed
+    consistent with this project or the open source license(s)
+    involved.
+```
+
+Subsequent developers who co-author or otherwise help shepherd the contribution in some way also add their own attestation so it's not unusual to end up with a contribution which looks like:
+
+```
+ Author: Nathen Harvey <nharvey@chef.io>
+
+ Committer: Adam Jacob <adam@chef.io>
+
+   Sprinkle extra delight on our DSL
+
+   Due to an oversight, we were only 96% delightful. Ensure the
+   delight now goes to 11.
+
+   Signed-off-by: Nathen Harvey <nharvey@chef.io>
+   Signed-off-by: Julia Cook <julia@ourcompany.com>
+   Signed-off-by: Adam Jacob <adam@chef.io>
+```
+
+For more information on the change see the Chef Blog post [Introducing Developer Certificate of Origin](https://blog.chef.io/2016/09/19/introducing-developer-certificate-of-origin/)
+
+#### DCO Sign-Off Methods
+
+The DCO requires a sign-off message in the following format appear on each commit in the pull request
+
+```
+Signed-off-by: Joe Smith <joe.smith@email.com>
+```
+
+The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit –amend -s**
```

Comment:
`--amend` I think, and maybe add ```(don't forget to force push, `git push -f`)``` and `If you need to retroactively apply DCO signoff on a multi-commit pull request, please ask for help in Slack`. I'm working on a better thing for the latter case but for now at least a signpost seems like a good step.